pyvista/core/filters.py

```diff
@@ -1129,6 +1129,41 @@ def warp_by_scalar(dataset, scalars=None, factor=1.0, normal=None,
             return
         return output
 
+    def warp_by_vector(dataset, vectors=None, factor=1.0, inplace=False):
+        """Warp the dataset's points by a point data vectors array's values.
+
+        This modifies point coordinates by moving points along point vectors by
+        the local vector times the scale factor.
+
+        A classical application of this transform is to visualize eigenmodes in mechanics.
+
+        Parameters
+        ----------
+        vectors : str, optional
+            Name of vector to warp by. Defaults to currently active vector.
+        factor : float, optional
+            A scaling factor that multiplies the vectors to warp by. Can be used to enhance the warping effect.
+        inplace : bool, optional
+            If True, the function will update the mesh in-place and return ``None``.
+        
+        Returns
+        -------
+        warped_mesh : mesh
+            The warped mesh resulting from the operation.
+
+        """
+        if vectors is None:
+            field, vectors = dataset.active_scalars_info
```

Comment:
make sure that the filter is capable of leveraging vectors that are set under the `vectors` attribute (i.e. have the name `"_vectors"`). This should be as simple as changing this line to 

```suggestion
            field, vectors = dataset.active_vectors_info
```

And then also be sure to check that the array is indeed a vector array (has three components), throwing an error if not.